spec/src/main/asciidoc/configprofile.asciidoc

```diff
@@ -44,11 +44,10 @@ If the property `mp.config.profile` is specified in multiple config sources, the
 ==== On Property level
 The configuration property that utilizes the Config Profile is called a "profile-specific" property. A "profile-specific" property name consists of the following sequence: `% <profile name>.<original property name>`.
 
-Conforming implementations are required to search for a configuration source with the highest ordinal (priority) that provides either the property name or the "profile-specific" property name.
+Conforming implementations are required to search all configuration sources that provide either the property name or the "profile-specific" property name.
 If the configuration source provides the "profile-specific" name, the value of the "profile-specific" property will be returned. If it doesn't contain the "profile-specific" name, the value of the plain property will be returned.
```

Comment:
After this line, I suggest adding:
The "profile-specific" concept only influence which value is to be surfaced per config source.

Reply:
Thanks, added.